src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42FlowRttSpec.groovy

```diff
@@ -953,9 +961,9 @@ class Server42FlowRttSpec extends HealthCheckSpecification {
         }
     }
 
-    def "getSwPairConnectedToS42ForVxlanFlow"(List<SwitchId> switchIdsConnectedToS42) {
+    def "getSwPairConnectedToS42ForVxlanFlowOnNsSwitch"(List<SwitchId> switchIdsConnectedToS42) {
```

Comment:
It's `non-Wb`, not `NS`, or pick NS. We may have other switches in the future. We may have virtual switches support vxlan soon. They will qualify based on the search below but they won't be NS